databook/physiology/np-ultra/np-ultra.md

```diff
@@ -0,0 +1,5 @@
+# Neuropixels Ultra
+
```

Comment:
We'll want to adjust this text but as a placeholder this is fine

Reply:
Left as is as placeholder